wear/src/main/java/io/homeassistant/companion/android/home/views/SensorUi.kt

```diff
@@ -21,13 +24,43 @@ fun SensorUi(
     basicSensor: SensorManager.BasicSensor,
     onSensorClicked: (String, Boolean) -> Unit,
 ) {
+    var checked = sensor?.enabled == true
+
+    val backgroundRequest =
+        rememberLauncherForActivityResult(ActivityResultContracts.RequestPermission()) {}
```

Comment:
In the callback here the sensor that was being toggled should be enabled if all required permissions are granted.

Reply:
This call back is really only used for background permissions as its the only one where we remove and request later so there isn't a need to check all other permissions since it will be the last one.

Reply:
But right now it means that the sensor won't be enabled after granting the required permission (because the callback isn't used), you have to tap it again to enable it.